Readme.md

```diff
@@ -1,21 +1,27 @@
-WWTD: Travis simulator - faster + no more waiting for build emails.<br/>
-Reads your .travis.yml and runs what travis would run (via rvm/rbenv/chruby).
+# WWTD: Travis simulator [![Build Status](https://travis-ci.org/grosser/wwtd.svg?branch=master)](https://travis-ci.org/grosser/wwtd)
+
+Reads your `.travis.yml` and runs what Travis would run using [rvm], [rbenv], or [chruby].  No more waiting for build emails!
 
 ![Results](assets/results.png?raw=true)
 
-Install
-=======
+[rvm]: https://github.com/rvm/rvm
+[rbenv]: https://github.com/rbenv/rbenv
+[chruby]: https://github.com/postmodern/chruby
```

Comment:
idk ... do we really need to link these 🤷‍♂

Reply:
Not strictly necessary, but they might be helpful to a beginner.  Also, I thought it would be a nice touch, since the blue links draw the eyes to the fact that wwtd supports all three (which is a major feature, IMO).

Reply:
yeah ... either way is fine 🤷‍♂